plugins/services/src/js/filters/__tests__/TaskNameTextFilter-test.js

```diff
@@ -0,0 +1,41 @@
+var DSLFilterList = require("#SRC/js/structs/DSLFilterList");
+var SearchDSL = require("#SRC/resources/grammar/SearchDSL.jison");
+var TaskNameTextFilter = require("../TaskNameTextFilter");
+var List = require("#SRC/js/structs/List");
+
+describe("TaskNameTextFilter", function() {
+  beforeEach(function() {
+    this.mockItems = [
+      {
+        name: "cassandra"
+      },
+      {
+        name: "node-1-server__1"
+      },
+      {
+        name: "node-0-server__2"
+      }
+    ];
+  });
+
+  it("Should match parts of task name", function() {
```

Comment:
Remove the `should` from all tests please. We are not following this pattern anymore.